.github/workflows/release.yaml

```diff
@@ -50,7 +48,9 @@ jobs:
       - name: Publish crates
         run: yes | make release
         env:
-          CRATES_TOKEN: ${{ secrets.CRATES_TOKEN }}
+          # https://doc.rust-lang.org/cargo/commands/cargo-publish.html#options
+          # argument for `--token` is specified with this environment variable
+          CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
```

Comment:
I think that `CRATES_TOKEN` and `CARGO_REGISTRY_TOKEN` are the same secret. `CRATES_TOKEN` was passed to `--token` when we were using `script/release.sh`.

https://github.com/cosmos/ibc-rs/blob/3703e298a3f408a16892738d3d4696cd24c2380c/scripts/release.sh#L51-L53

So we can remove the duplicate secret from the repo.